jsapp/js/account/plan.component.tsx

```diff
@@ -212,17 +226,32 @@ export default function Plan() {
 
   const isSubscribedProduct = useCallback(
     (product: Price) => {
-      if (hasActiveSubscription !== undefined) {
-        if (!product.prices.unit_amount && !hasActiveSubscription) {
+      if (
+        !product.prices.unit_amount &&
+        state.intervalFilter === 'year' &&
```

Comment:
Getting better; now the 'your plan' banner shows up for the Community plan when it's annual, but not when showing monthly plans. I'm guessing it's related to this line.

Reply:
Yup! I just found this while testing and pushed a change to fix it

Reply:
Unfortunately, it's still only showing for the annual plan. I'd suggest reverting the last commit and just removing line 231 (`state.intervalFilter === 'year' &&`).